src/components/MovieList/MovieList.tsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
 import {MovieCard} from '../MovieCard';
 import styles from './MovieList.module.css';
-import {useSelector} from 'react-redux';
-import {MovieState} from '../../redux/reducer';
-import {RootReducer} from '../../store';
+import {Movie} from "../../redux/types";
 
-export const MovieList = () => {
-    const {moviesList} = useSelector<RootReducer, MovieState>((state) => state.movies);
+interface MoviesListProps {
+    moviesList: Movie[]
+}
 
+export const MovieList = ({moviesList}: MoviesListProps) => {
     return (
-        <div className={styles.list}>
-            {moviesList.map((movie) =>
-                <MovieCard
-                    image={movie.poster_path}
-                    title={movie.title}
-                    subtitle={movie.tagline}
-                    year={String(new Date(movie.release_date).getFullYear())}
-                    key={movie.id}
-                    id={movie.id}
-                />
-            )}
+        <div className={styles.content}>
+            {
+                moviesList.length ?
+                    <div className={styles.list}>
+                        { moviesList.map((movie) =>
+                            <MovieCard
+                                image={movie.poster_path}
+                                title={movie.title}
+                                subtitle={movie.tagline}
+                                year={String(new Date(movie.release_date).getFullYear())}
```

Comment:
`String(new Date(movie.release_date).getFullYear())`
can be rewritten to
`${new Date(movie.release_date).getFullYear()}`